### ISSUE TITLE
Moving elements to the last position in a group needs to be very exact

### ISSUE THREAD

**Issue by jancborchardt** [penpot/penpot#889]
**Is your feature request related to a problem? Please describe.**
If you want to move an element/layer to the last position in a group is very finicky. Especially when there’s a subgroup.

**Describe the solution you'd like**
Instead of just this thin white line which gets shown as a drag receiving indicator, it would be nice if there’s some thicker line (possibly also that nice green main color) which is more welcoming to the dragged layer. ;)

**Additional context**
![Moving layers needs to be exact](https://user-images.githubusercontent.com/925062/116714421-10824380-a9d6-11eb-95ae-6240c820d2e6.png)

**Comment by elhombretecla** [comment]
Yes, sometimes is difficult to organize layers between groups. We will review this :)

Taiga issue [https://tree.taiga.io/project/penpot/issue/1630](https://tree.taiga.io/project/penpot/issue/1630)
